qt/scientific_interfaces/Indirect/IndirectFitOutput.cpp

```diff
@@ -217,6 +217,59 @@ void renameResult(WorkspaceGroup_sptr resultWorkspace,
     renameResultWithoutSpectra(resultWorkspace, fitDataBegin, fitDataEnd);
 }
 
+std::vector<std::string> strsplit(const std::string &string_in,
+                                  const std::string &delim) {
+  std::vector<std::string> split_vect;
+  iter_split(split_vect, string_in, boost::algorithm::first_finder(delim));
+  return split_vect;
+}
+
+std::vector<std::string> srange(const std::string &string_range) {
+  // e.g. string_range = "0-2+4"
+  // output = "0", "1", "2", "4"
+  std::vector<std::string> limits = strsplit(string_range, "-");
+  if (limits.size() == 1) {
+    return limits;
+  } else {
+    std::vector<std::string> out;
+    for (int i = boost::lexical_cast<int>(limits[0]);
+         i != boost::lexical_cast<int>(limits[1]) + 1; ++i) {
+      out.push_back(boost::lexical_cast<std::string>(i));
+    }
+    return out;
+  }
+}
+
+void renameFitWorkspace(WorkspaceGroup_sptr resultGroup,
+                        WorkspaceGroup_sptr resultWorkspace) {
+  // rename the fit data workspaces using result workspace names
+  std::size_t indexWS = 0; // index of fit workspace in resultGroup
+  const auto numRuns = static_cast<std::size_t>(resultWorkspace->size());
+  for (std::size_t index = 0; index != numRuns; ++index) {
+    const auto ws = resultWorkspace->getItem(index);
+    const auto name = ws->getName();
+    // extract relevent parts of the name
+    const auto nameParts = strsplit(name, "_iqt");
+    std::string runstr = nameParts[0];
```

Comment:
A lot of these variables should be `const auto`.

(`const` as they are not mutated, `auto` because [AAA](https://herbsutter.com/2013/08/12/gotw-94-solution-aaa-style-almost-always-auto/))